lib/analyzer/apt-analyzer.ts

```diff
@@ -3,12 +3,25 @@ import { AnalyzerPkg } from "./types";
 
 export { analyze };
 
-async function analyze(targetImage: string, options?: DockerOptions) {
-  const docker = new Docker(targetImage, options);
-  const dpkgFile = (await docker.catSafe("/var/lib/dpkg/status")).stdout;
+const APT_DPKG_STATUS = "/var/lib/dpkg/status";
+const APT_EXT_STATES = "/var/lib/apt/extended_states";
+
+const APT_PKGFILES = [APT_DPKG_STATUS, APT_EXT_STATES];
+
+export { APT_PKGFILES };
+
+async function analyze(
+  targetImage: string,
+  options?: DockerOptions,
+  files?: {} | void,
```

Comment:
```
files? : { [key: string]: string }
```

is probably more like what you want, depending on the exact format.

Reply:
Done